src/main/scala/subsystem/BaseSubsystem.scala

```diff
@@ -134,7 +134,21 @@ abstract class BaseSubsystem(implicit p: Parameters) extends BareSubsystem
     }
   }
 
-  lazy val logicalTreeNode = new SubsystemLogicalTreeNode()
+  lazy val logicalTreeNode = {
+    val subsystemLogicalTreeNode = new SubsystemLogicalTreeNode()
+    val builtInDevices = Seq(
+      pbus.builtInDevices,
+      fbus.builtInDevices,
+      mbus.builtInDevices,
+      cbus.builtInDevices
+    )
+    for (builtIn <- builtInDevices) {
+      builtIn.errorOpt.foreach { error =>
```

Comment:
Since this has taken the trouble to do "the hard part" here I am wondering if we shouldn't extend this PR to do the same for builtIn.zeroOpt. I can push to this branch to add that...

Reply:
I will push a change that adds the OMZero stuff but won't touch this part of the code since it's still under discussion